dockerfiles/Dockerfile

```diff
@@ -56,10 +43,18 @@ ARG ROBOTOLOGY_SUPERBUILD_INSTALL_DIR=/usr/local
 # Set up git (required by superbuild)
 RUN git config --global user.name "GitHub Actions" && \
     git config --global user.email "actions@github.com"
+    
+# Install dependencies
+RUN git clone https://github.com/robotology/robotology-superbuild.git --depth 1 --branch ${ROBOTOLOGY_SUPERBUILD_RELEASE} && \
+    robotology-superbuild/scripts/install_apt_dependencies.sh
```

Comment:
We do rely on the `robotology-superbuild` infrastructure to install most of the dependencies.